jest.config.ts

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
I find it frustrating when there are too many config files in the root as they sort of pollute the root dir and makes it hard to locate. What do you think of having a config folder and we put our webpack, babel, jest and tsconfig in it? Don't have to be done in this PR, but what are your thoughts?

Reply:
Sounds good i'll i implement it on this branch